openapi/openapi.yaml

```
@@ -375,8 +375,7 @@ tags:
       amounts, they do not modify the transaction amount. Fees help to
       describe each part of the transaction amount.
 
-      This API is experimental and not guaranteed to be backward
-      compatible.
+      > **Important:** This operation is experimental and may change.
```

```suggestion
      > **Important:** These operations are experimental and may change.
```

There are several `Fees` operations